src/react/components/common/filePicker/filePicker.tsx

```diff
@@ -8,7 +8,7 @@ import HtmlFileReader from "../../../../common/htmlFileReader";
  * @member onError - Function to call on file picking error
  */
 interface IFilePickerProps {
-    onChange: (sender: SyntheticEvent, fileText: string | ArrayBuffer) => void;
+    onChange: (sender: SyntheticEvent, fileText: string | ArrayBuffer | File) => void;
```

Comment:
Similar to above - it looks like `fileText` here will always be your new `fileInfo` vs every returning only the string/arraybuffer.